app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java

```diff
@@ -50,7 +50,6 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             public void onRefresh() {
                 if (haveNetworkConnection()) {
                     DataDownload download = new DataDownload();
-                    dbSingleton.clearDatabase(DbContract.Sponsors.TABLE_NAME);
```

Comment:
In api/processor/*, for every success we are clearing the respective tables of database and adding new values.So, for every onRefresh(),its will clear the tables only when we have completed fetching that data else retain it....Other wise it will show blank screen as reported in issue #117

Take the case....network is not available and u deleted the table...now so nothing to show